crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/FullAssessmentAvailabilityService.java

```diff
@@ -15,64 +14,57 @@
 @Service
 public class FullAssessmentAvailabilityService {
 
-    public void processFullAssessmentAvailable(final MeansAssessmentRequestDTO requestDTO,
-                                               final ApiMeansAssessmentResponse meansAssessmentResponse) {
+    public boolean getFullAssessmentAvailable(final CaseType caseType,
```

Comment:
Can we make this `isFullAssessmentAvailable()`?

Reply:
I think this could equally apply to a few other the other methods in the class too.